tasks/update/task.yaml

```diff
@@ -2,37 +2,41 @@ summary: Update Linode image.
 
 execute: |
 
-    case $SPREAD_REBOOT in
+    if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
+        dnf upgrade -y
```

Comment:
Couldn't we do some cleanup after rebooting into the new version as we do on ubuntu?